src/edu/wright/cs/raiderplanner/controller/MainController.java

```diff
@@ -2,7 +2,7 @@
  * Copyright (C) 2017 - Benjamin Dickson, Andrew Odintsov, Zilvinas Ceikauskas,
  * Bijan Ghasemi Afshar, Amila Dias
  *
- * Copyright (C) 2018 - Clayton D. Terrill
```

Comment:
Why is Clayton's copyright assertion removed? It doesn't seem like it should be removed.